pkg/pgmodel/metrics/ingest.go

```diff
@@ -167,6 +167,25 @@ var (
 			Help:      "Total number of requests to ingestor.",
 		}, []string{"type", "code"},
 	)
+	InsertablesReceivedTotal = prometheus.NewCounterVec(
```

Comment:
We dont need this, since `IngestorItems` handles it.

Reply:
If we add this, then this has to be done across samples as well, which it does not. It earlier used to, but there wasn't a need and hence it was removed.